teamsupport/models.py

```diff
@@ -1,19 +1,31 @@
 from property_caching import cached_property
 from querylist import QueryList
 
+from teamsupport.services import TeamSupportService
 from teamsupport.errors import MissingArgumentError
 
 
+TICKET_STATUS_NEW = '212203'
+TICKET_TYPE_SUPPORT = '35731'
```

Comment:
assuming I understand this correctly: that these are IDs that are unique to our status values, in which case I agree with @stefanor